install.sh

```diff
@@ -58,6 +97,14 @@ do
   esac
 done
 
+if [ $silent ]; then
+  # backup/new by default
+  test -w "$HOME/$CONFIG_FILE" &&
```

Comment:
It might make sense to move this block into a function, since it is used twice...